### ISSUE TITLE
failing tests after django 2.2 upgrade

### ISSUE THREAD

**Issue by thenewguy** [JamesRamm/longclaw#258]
Tests are failing:

```
vagrant@ubuntu-xenial:/vagrant$ python3 manage.py test --verbosity=3 longclaw.core
Creating test database for alias 'default' ('file:memorydb_default?mode=memory&cache=shared')...
Operations to perform:
  Synchronize unmigrated apps: core, django_extensions, messages, modelcluster, productrequests, rest_framework, staticfiles, stats, wagtailmodeladmin, wagtailsettings, wagtailsites, wagtailsnippets
  Apply all migrations: admin, auth, basket, checkout, configuration, contenttypes, orders, products, sessions, shipping, sites, taggit, testproducts, trivialrates, wagtailadmin, wagtailcore, wagtaildocs, wagtailembeds, wagtailforms, wagtailimages, wagtailredirects, wagtailsearch, wagtailusers
Running pre-migrate handlers for application admin
Running pre-migrate handlers for application auth
Running pre-migrate handlers for application contenttypes
Running pre-migrate handlers for application sites
Running pre-migrate handlers for application sessions
Running pre-migrate handlers for application wagtailforms
Running pre-migrate handlers for application wagtailredirects
Running pre-migrate handlers for application wagtailembeds
Running pre-migrate handlers for application wagtailusers
Running pre-migrate handlers for application wagtailsnippets
Running pre-migrate handlers for application wagtaildocs
Running pre-migrate handlers for application wagtailimages
Running pre-migrate handlers for application wagtailsearch
Running pre-migrate handlers for application wagtailadmin
Running pre-migrate handlers for application wagtailcore
Running pre-migrate handlers for application wagtailsettings
Running pre-migrate handlers for application modelcluster
Running pre-migrate handlers for application taggit
Running pre-migrate handlers for application django_extensions
Running pre-migrate handlers for application core
Running pre-migrate handlers for application configuration
Running pre-migrate handlers for application shipping
Running pre-migrate handlers for application products
Running pre-migrate handlers for application orders
Running pre-migrate handlers for application checkout
Running pre-migrate handlers for application basket
Running pre-migrate handlers for application stats
Running pre-migrate handlers for application productrequests
Running pre-migrate handlers for application testproducts
Running pre-migrate handlers for application trivialrates
Synchronizing apps without migrations:
  Creating tables...
    Creating table productrequests_productrequest
    Running deferred SQL...
Traceback (most recent call last):
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/db/backends/utils.py", line 82, in _execute
    return self.cursor.execute(sql)
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/db/backends/sqlite3/base.py", line 381, in execute
    return Database.Cursor.execute(self, query)
sqlite3.OperationalError: no such table: testproducts_productvariant

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "manage.py", line 12, in <module>
    execute_from_command_line(sys.argv)
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/core/management/commands/test.py", line 23, in run_from_argv
    super().run_from_argv(argv)
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/core/management/commands/test.py", line 53, in handle
    failures = test_runner.run_tests(test_labels)
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/test/runner.py", line 629, in run_tests
    old_config = self.setup_databases(aliases=databases)
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/test/runner.py", line 554, in setup_databases
    self.parallel, **kwargs
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/test/utils.py", line 174, in setup_databases
    serialize=connection.settings_dict.get('TEST', {}).get('SERIALIZE', True),
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/db/backends/base/creation.py", line 72, in create_test_db
    run_syncdb=True,
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/core/management/__init__.py", line 148, in call_command
    return command.execute(*args, **defaults)
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/core/management/base.py", line 83, in wrapped
    res = handle_func(*args, **kwargs)
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/core/management/commands/migrate.py", line 203, in handle
    self.sync_apps(connection, executor.loader.unmigrated_apps)
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/core/management/commands/migrate.py", line 341, in sync_apps
    self.stdout.write("    Running deferred SQL...\n")
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/db/backends/sqlite3/schema.py", line 34, in __exit__
    self.connection.check_constraints()
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/db/backends/sqlite3/base.py", line 341, in check_constraints
    column_name, referenced_column_name,
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/db/backends/utils.py", line 76, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/db/utils.py", line 89, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/db/backends/utils.py", line 82, in _execute
    return self.cursor.execute(sql)
  File "/home/vagrant/.local/lib/python3.5/site-packages/django/db/backends/sqlite3/base.py", line 381, in execute
    return Database.Cursor.execute(self, query)
django.db.utils.OperationalError: no such table: testproducts_productvariant

```

**Comment by thenewguy** [comment]
This issue can be closed when PR #259 is merged. All tests passing

**Comment by JamesRamm** [comment]
closed by #259
